requirements.txt

```diff
@@ -6,8 +6,8 @@ Jinja2==3.1.3
 jira==3.4.1
 jproperties==2.1.1
 jsonpath_ng==1.6.0
-lxml==4.9.1
-numpy==1.22.4
+lxml==4.9.4
+numpy==1.26.4
```

Comment:
We don't actually need numpy, so dropped it in https://github.com/cilt-uct/Brightspace-migrate-from-sakai/commit/33a1e813da1fbdbdaf4aade23c62873aa2fc5c58